facebook/react-native
# Add vertical ScrollView content offset on Android

This is the (evil) twin commit to:
https://github.com/facebook/react-native/pull/12502

Fixes that contentOffset is not available on
vertical scroll-views on Android.

## Test Plan
* Create an Android component with a ScrollView. Add enough
  items to fill beyond the visible screen. Set the content
  offset to some value that makes the content scroll. Verify
  that it snaps to the y-axis as expected.
* Remove items so screen is no longer filled, keep content
  offset parameter. Scroll view still works but does not scroll.
* Set only one parameter to content-offset (x || y) -
  parameters are now ignored since they must be a pair.
* Remove contentOffset, scrollView still works as before.
